src/query.rs

```diff
@@ -75,6 +78,13 @@ pub fn q_once<'sqlite, 'schema, 'query, T, U>
 
     algebrized.apply_limit(limit.into());
 
+    // Because this is q_once, we can check that all of our `:in` variables are bound at this point.
+    // If they aren't, the user has made an error -- perhaps writing the wrong variable in `:in`, or
+    // not binding in the `QueryInput`.
+    let unbound = algebrized.unbound_variables();
```

Comment:
I did this because I wrote a test that failed because I typoed the `:in` variable :)